packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelSummary.tsx

```diff
@@ -291,7 +292,12 @@ export function TransferPanelSummary({
         <span className="w-2/5 font-light">You’re moving</span>
         <div className="flex w-3/5 flex-row justify-between">
           <span>
-            {formatAmount(amount, { symbol: token?.symbol || 'ETH' })}
+            {formatAmount(amount, {
+              symbol: sanitizeTokenSymbol(token?.symbol || 'ETH', {
+                erc20L1Address: token?.address,
+                chain: app.isDepositMode ? l1.network : l2.network
+              })
+            })}
```

Comment:
nit: let's create a memoized property for this, similar to what we have in other components

```
  const tokenSymbol = useMemo(
    () =>
      token
        ? sanitizeTokenSymbol(token.symbol, {
            erc20L1Address: token.address,
            chain: app.isDepositMode ? l1Network : l2Network
          })
        : 'ETH',
    [token, app.isDepositMode, l2Network, l1Network]
  )
```

Reply:
Done wherever applicable.